src/pattern-library/components/patterns/prototype/TabbedDialogPage.tsx

```diff
@@ -207,6 +206,7 @@ function TabbedSharePanel() {
             variant="custom"
             title="Share annotations"
             onClose={() => setPanelOpen(false)}
+            transitionComponent={Slider}
```

Comment:
OK, so now transitions work, but adding this introduces a regression with the keyboard navigation vis-a-vis the tabs. I'll try to debug a little further.